cmd/config/set/hub_rpc.go

```diff
@@ -0,0 +1,21 @@
+package set
+
+import (
+	"github.com/dymensionxyz/roller/config"
+	"github.com/dymensionxyz/roller/relayer"
+	"github.com/dymensionxyz/roller/sequencer"
+	"github.com/dymensionxyz/roller/utils"
+)
+
+func setHubRPC(rlpCfg config.RollappConfig, value string) error {
+	rlpCfg.HubData.RPC_URL = value
```

Comment:
We should probably also update the API_URL and any other URLS if exists (so better call it maybe update hub-node):

e.g config.toml:

config.toml:  API_URL = "https://froopyland.api.silknodes.io:443"
config.toml:  RPC_URL = "https://froopyland.rpc.silknodes.io:443"

Reply:
disagree, not necessarily related